Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:20-slim AS base
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+
+FROM base AS build
+## needed for node-gyp to build
+RUN apt-get update && \
+    apt-get install -y python3 make gcc g++
+COPY . /usr/src/app
+WORKDIR /usr/src/app
+RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --frozen-lockfile
```

Comment:
Should we use `--prod` option to avoid installing dev dependencies?

Reply:
The prod option is on line 14 in the deploy command:
RUN pnpm deploy --filter=commonwealth --prod /prod/commonwealth

We need to initially install dev dependencies to build the client bundle